tests/tap.sh

```diff
@@ -31,7 +32,12 @@ elif [ ${1::4} == "nek-" ]; then
       allargs+=("$(awk -v i="$i" '/C_TESTARGS/,/\n/{j++}j==i+1{print; exit}' examples/nek/bps/${1:4}.usr* | cut -d\  -f2- )")
     done
 elif [ ${1::7} == "fluids-" ]; then
-    allargs=$(grep -F //TESTARGS examples/fluids/${1:7}.c* | cut -d\  -f2- )
+    # get all test configurations
+    numconfig=$(grep -F //TESTARGS examples/fluids/${1:7}.c* | wc -l)
+    for ((i=0;i<${numconfig};++i)); do
+      suffices+=("$(awk -v i="$i" '/\/\/TESTARGS/,/\n/{j++}j==i+1{print; exit}' examples/fluids/${1:7}.c | cut -d\  -f2- | cut -d ";" -f 1 | cut -d "=" -f 2 )")
```

Comment:
Just to be clear, this was done because initially we said to to support CI processing for multiple test configurations (i.e., multiple `//TESTARGS` lines with different suffices) which produce some numerical output. But now, my current understanding of the latest suggestion is that the check for consistency with the reference solution is going to happen in the user code (I am on it), hence all this machinery in `tap.sh` will be gone, and `tap` will just check if the comparison has produced some output or not, as for all other examples

Reply:
If this code is debugged, we'll probably use it eventually.

Reply:
Yes it is and I had tested it. How to preserve it for future reference?

Reply:
These edits to `tap.sh` to process multiple tests with output are all contained in one commit ([a0c2d49](https://github.com/CEED/libCEED/pull/516/commits/a0c2d496e17e245ba9197a78c69f3b0c81057b90)), so it will be easy to retrieve that for future reference if we don't squash and merge

Reply:
That commit could be cherry-picked, but I don't mind if it just rides in with this branch.